src/mumble/CoreAudio.h

```diff
@@ -10,6 +10,9 @@
 #include "AudioOutput.h"
 
 #include <AudioToolbox/AudioToolbox.h>
+#include <Carbon/Carbon.h>
+
+#include "Global.h"
```

Comment:
Why did you move the includes to the header?

Reply:
Because I move the declaration of two classes to the header file and DeferInit is defined in Global.h. Actually Carbon.h is not necessary to import from the header file.

Reply:
All clear, thank you very much for your contribution!